python/Ganga/Utility/Runtime.py

```diff
@@ -257,12 +257,14 @@ def loadPlugins(environment):
             logger.error('problems with loading Named Templates for %s', n)
             logger.error('Reason: %s' % str(err))
 
-    for r in allRuntimes.values():
+    for n, r in zip(allRuntimes.keys(), allRuntimes.values()):
```

Comment:
Following coding style above so I'd rather stick to what's already used for sake of readability.